test/cssBaseTest.js

```diff
@@ -46,4 +64,17 @@ describe("css-base", function() {
 		}]);
 		m.toString().should.be.eql("body { a: 1; }\n/*# sourceURL=webpack://./path/to/test.scss */\n/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJmaWxlIjoidGVzdC5zY3NzIiwic291cmNlcyI6WyIuL3BhdGgvdG8vdGVzdC5zY3NzIl0sIm1hcHBpbmdzIjoiQUFBQTsiLCJzb3VyY2VSb290Ijoid2VicGFjazovLyJ9 */");
 	});
+	it("should toString without source mapping if btoa not avalibale", function() {
```

Comment:
Why CSS should be extracted without source mapping if btoa not avalibale?
This is BREAKING CHANGE. In node <= 14 the CSS  has not the inline source map.
If btoa not avalibale then shoul be used the `Buffer.from(string).toString('base64')` instead of `btoa(string)`.

Reply:
ok, I understand.

P.S: I use the css-loader runtime in a webpack plugin to generate inline source-map. In node 16 it works, but not in older node.